src/translator/html.cpp

```diff
@@ -31,12 +31,12 @@ void encodeEntities(marian::string_view const &input, std::string &output) {
       // case ???:
       //   output.append("&nbsp;");
       //   break;
-      case '"':
-        output.append("&quot;");
-        break;
-      case '\'':
-        output.append("&apos;");
-        break;
+      // case '"':
+      //   output.append("&quot;");
+      //   break;
+      // case '\'':
+      //   output.append("&apos;");
+      //   break;
```

Comment:
What's happening here? Is HTML subword splitting QE influencing/motivating HTML render?

Reply:
I initially started escaping the single and double quote to allow for the text to work with html attributes. But I'm not adding text to attributes any more, so there is no need to support those in the escaping routine.